### DISCUSSION TITLE
How to deploy static site?

### DISCUSSION THREAD

**Discussion by extraric** [community/community#110050] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't really understand what have to do to start serving my project site with simple index.html. 
![image](https://github.com/community/community/assets/45036037/b7e8f28e-613e-4d34-a679-051a74999b10)

I set up yesterday and nothing changes, should I run some publishing action?
The previous similar site I hadn't set up any workflow as I remember.

**Comment by SaraRasoulian** [comment]
could you please share your repository that you are trying to deploy?

**Reply by extraric** [reply]
Sorry, I only have two here, this is the subject: https://github.com/extraric/Balatonkenese

**Reply by SaraRasoulian** [reply, marked as answer]
I created a fork of your repository and deployed it: https://sararasoulian.github.io/Balatonkenese/
here is the fork: https://github.com/SaraRasoulian/Balatonkenese
there wasn't any issue and you don't need any workflow

please try again, maybe you forgot to click on save button


![image](https://github.com/community/community/assets/51083712/00747383-1ff0-47b2-9936-65fbf58baed2)

**Reply by SaraRasoulian** [reply]
I changed the links in index.html file, like this:
```
<img class="alaprajz" src="images/alaprajz.jpg" alt="alaprajz">
```
the image wasn't showing at first
any way I'll remove the fork later

**Reply by extraric** [reply]
Changed back to None, saved, and again to master, /(root), and Save, to get this nice message:
![image](https://github.com/community/community/assets/45036037/a7ab3783-07a0-41b8-a467-2d7e92a74214)
😄 

The links are good, it's relative to root, I want to server on custom domain, there will be OK.
Pushed new commit, and now it is serving fine, thank you for support!
